### ISSUE TITLE
IMEX is not imported to new database

### ISSUE THREAD

**Issue by cenx1** [A9G-Data-Droid/AccessDB-BuildFromSource#3]
Having a database exported using https://github.com/joyfullservice that contains Import/Export templates (IMEX located in tables __MSysIMEXSpecs__ and __MSYSIMEXColumns__), does not get added to the compiled database.

**Comment by A9G-Data-Droid** [comment]
Correct, the code for exporting tables specifically skips anything that starts with "MSys" OR "~". This comes from the historical upstream project. The reason is that system tables are not editable. If you want to save Import/Export specifications then additional code needs to be added to handle that.

**Comment by cenx1** [comment]
OK, I see. I will try and look into this.

**Comment by A9G-Data-Droid** [comment]
I think this has been resolved in upstream. Please use the compile feature in the latest version of the version control add-in: https://github.com/joyfullservice/msaccess-vcs-integration
